susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -127,16 +129,39 @@ bootstrap_repo:
       - ([ {{ bootstrap_repo_exists }} = "True" ])
 {%- endif %}
 
+{%- if grains['os_family'] == 'RedHat' %}
+trust_res_gpg_key:
+  cmd.run:
+    - name: rpm --import https://{{ salt['pillar.get']('mgr_server') }}/pub/{{ salt['pillar.get']('gpgkeys:res:file') }}
+    - unless: rpm -q {{ salt['pillar.get']('gpgkeys:res:name') }}
+    - runas: root
+{%- elif grains['os_family'] == 'Debian' %}
+install_gnupg_debian:
+  pkg.latest:
+    - pkgs:
+      - gnupg
+{%- endif %}
+
```

Comment:
This duplicate states. One line below we include "channels/gpg-keys.sls". That is the place where it should happen and afaik this is already implemented.

Reply:
I think it was side effect with rebasing the commit, fixed